### DISCUSSION TITLE
New release version

### DISCUSSION THREAD

**Discussion by mohrlex83** [pawelsalawa/sqlitestudio#4544] [General]
When new release?

**Comment by pawelsalawa** [comment]
When all items in 3.4.0 milestone are done: https://github.com/pawelsalawa/sqlitestudio/milestone/4
I cannot tell when exactly it will be. Maybe a month, maybe 2, maybe more...? Depends on how much effort will be needed for these issues.

**Comment by pawelsalawa** [comment]
Well. It just happened :)

**Reply by JrgMyr** [reply]
Great!
Congratulations.
I'll anounce it on Reddit.
